util/oidc/oidc_test.go

```diff
@@ -179,3 +185,49 @@ func TestIsValidRedirect(t *testing.T) {
 		})
 	}
 }
+
+func TestGenerateAppState(t *testing.T) {
+	pass := "test-server-signature"
```

Comment:
Should we use a password generated by `MakeSignature`, just so it matches the use case more closely?

otoh I can see the downside of introducing random input for a non-fuzzing test.

Reply:
done